src/geometry/euclidean.lean

```diff
@@ -5,8 +5,10 @@ Author: Joseph Myers.
 -/
 import analysis.normed_space.real_inner_product
 import analysis.normed_space.add_torsor
+import tactic.apply_fun
```

Comment:
I think we should just add `tactic.apply_fun` to the default tactic imports. @urkud do you agree?

Reply:
Why do you ask me? AFAIR I never touched the default tactic imports.

Reply:
Hmm... sorry. Some connection went wrong in my brain. @digama0 you're the boss. What's the policy with adding stuff to the default tactic imports?

Reply:
[`tactic.apply_fun` is already in `src/tactic/default.lean`](https://github.com/leanprover-community/mathlib/blob/master/src/tactic/default.lean#L25). But I don't think we want to `import tactic` in all other files in mathlib, so what exactly do you have in mind?

Reply:
Probably @jcommelin intended to suggest we import `tactic.apply_fun` already in `src/tactic/basic.lean`. That seems very reasonable.

Reply:
Ah, that makes sense. But I'm not so sure that's a good idea either. `tactic.apply_fun` imports `tactic.monotonicity`, which in turn imports `algebra.order_functions`, etc.

Reply:
I'll mark this suggestion as resolved (i.e. no change necessary), then.